ipfs-cluster-ctl/main.go

```diff
@@ -272,6 +273,15 @@ peers should pin this content.
 							Name:  "no-status, ns",
 							Usage: "Prevents fetching pin status after pinning (faster, quieter)",
 						},
+						cli.BoolFlag{
+							Name:  "wait, w",
+							Usage: "Waits for all nodes to report a status of pinned before returning",
+						},
+						cli.DurationFlag{
+							Name:  "wait-timeout, wt",
+							Value: 0,
+							Usage: "Specify how long (in seconds) you are willing to wait for pinned status (default is indefinitely)",
```

Comment:
We have a sharness tests that limits the width of help lines to 120 chars (when doing `ipfs-cluster-ctl -h`). This probably hits it.